src/core/tests/Test_ConfigurePatchingProcessor.py

```diff
@@ -129,12 +129,12 @@ def test_operation_success_for_installation_request_with_configure_patching(self
         self.assertTrue(substatus_file_data_patch_metadata_summary["shouldReportToHealthStore"])
         runtime.stop()
 
-    def test_operation_fail_for_configure_patching_agent_incompatible(self):
+    def test_operation_fail_for_configure_patching_telemetry_not_supported(self):
         argument_composer = ArgumentComposer()
         argument_composer.operation = Constants.CONFIGURE_PATCHING
         argument_composer.patch_mode = Constants.PatchModes.AUTOMATIC_BY_PLATFORM
         argument_composer.events_folder = None
-        runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.APT)
+        runtime = RuntimeCompositor(argument_composer.get_composed_arguments(env_settings=dict(telemetrySupported=False)), True, Constants.APT)
```

Comment:
env_settings=dict(telemetrySupported=False) this will only set telemetrySupported in env_settings and no value for the rest (i.e. config_folder, status_folder, etc)